linera-views/tests/queueview_tests.rs

```diff
@@ -43,15 +41,13 @@ async fn queue_view_mutability_check() {
                     new_vector.push(val);
                 }
             }
-            if choice == 1 {
+            if choice == 1 && count > 0 {
                 // deleting some entries
-                if count > 0 {
-                    let n_remove = rng.gen_range(0..count);
-                    for _ in 0..n_remove {
-                        view.queue.delete_front();
-                        // slow but we do not care for tests.
-                        new_vector.remove(0);
-                    }
+                let n_remove = rng.gen_range(0..count);
```

Comment:
Should this be `0..=count` or do we want to avoid removing all of them?

Reply:
Yes, you are right. Thank you.